### PR TITLE
Okazaki - Pathology Detail - P1

### PR DESCRIPTION
**Pull request by wwysoc2**
https://gdc-ctds.atlassian.net/browse/DICT-515

### CODE REVIEW

**Review by clsackett — APPROVED** [review]

File: src/gdcdictionary/schemas/pathology_detail.yaml [hunk 1]
```diff
@@ -576,6 +576,8 @@ properties:
       - Positive
       - Unknown
       - Not Reported
+    deprecated_enum:
+      - Indeterminant
```
Comment (wteouchicago) [inline src/gdcdictionary/schemas/pathology_detail.yaml]: i didn't see this change specified in the ticket.

File: src/gdcdictionary/schemas/pathology_detail.yaml [hunk 2]
```diff
@@ -497,8 +499,6 @@ properties:
       - Supraclavicular
       - Unknown
       - Not Reported
-    deprecated_enum:
-      - Cervical
```
Comment (wteouchicago) [inline src/gdcdictionary/schemas/pathology_detail.yaml]: i didn't see this change (undeprecating?) in the ticket.

**Comment by wteouchicago** [comment]
i didn't see:
- deprecation of `extrascleral_extension`
- adding `extrascleral_extension_present` and its enums
- adding `timepoint_category` and its enums

File: src/gdcdictionary/schemas/pathology_detail.yaml (hunk 1 above)
Comment (bullenc) [reply src/gdcdictionary/schemas/pathology_detail.yaml]: Fixed

File: src/gdcdictionary/schemas/pathology_detail.yaml (hunk 2 above)
Comment (bullenc) [reply src/gdcdictionary/schemas/pathology_detail.yaml]: Fixed; python + some packages were updated and the generating scripts are behaving weird, we'll probably have a few more of these 😞

**Comment by bullenc** [comment]
> i didn't see:
> 
>     * deprecation of `extrascleral_extension`
> 
>     * adding `extrascleral_extension_present` and its enums
> 
>     * adding `timepoint_category` and its enums

@wteouchicago  updated ticket to reflect that these will be Phase2 changes

**Review by wteouchicago — APPROVED** [review]

**Review by ebusygina — APPROVED** [review]

**Review by stilesj-uchicago — COMMENTED** [review]
Am I missing the `tumor_basal_diameter` that was defined in the ticket for PHASE 1?

**Comment by wwysoc2** [comment]
@stilesj-uchicago Somehow that seemed to slip by, but it should have been in Phase 2 (new props come in p2), so I've modified the JIRA ticket.

**Review by stilesj-uchicago — APPROVED** [review]

**Pull request merged**